plugins/python-build/share/python-build/pypy-2.3.1-src

```diff
@@ -1,4 +1,4 @@
 #require_gcc
 prefer_openssl11
 install_package "openssl-1.1.0j" "https://www.openssl.org/source/old/1.1.0/openssl-1.1.0j.tar.gz#31bec6c203ce1a8e93d5994f4ed304c63ccf07676118b6634edded12ad1b3246" mac_openssl --if has_broken_mac_openssl
-install_package "pypy-pypy-32f35069a16d" "https://bitbucket.org/pypy/pypy/get/release-2.3.1.tar.bz2#3fd10d97c0177c33ed358a78eb26f5bf1f91b266af853564b1a9d8c310a1e439" "pypy_builder" verify_py27 ensurepip
+install_package "pypy-pypy-32f35069a16d" "https://downloads.python.org/pypy/pypy-2.3.1-src.tar.bz2#6b10efab2be5502b475a25181870616e9ff83296b225aa2c7a7ec617c0ff78ef" "pypy_builder" verify_py27 ensurepip
```

Comment:
seems weird that the checksum would change, why isnt the binary the same?

Reply:
No idea. Maybe it was repackaged with different time stamps or something. Ask the project members if you want to know.

All that matters for Pyenv is that the new site is still an official one -- so the downloads are presumed trustworthy -- and since Pyenv checks the signatures, we have to update them if they change.